client/app/match/match.js

```diff
@@ -1,20 +1,13 @@
 angular.module( 'dinnerDaddy.match', ['dinnerDaddy.services'] )
 
-.controller( 'MatchController', function( $scope, Match, Auth, Session, Socket, Restaurant, $cookies) {
+.controller( 'MatchController', function( $scope, $rootScope, Match, Auth, Session, Socket, Restaurant, $cookies, $window) {
   $scope.session = {};
   $scope.user = {};
```

Comment:
We already have user on the rootscope, so we don't need this, or to set the user.name using cookies.

In another PR, we should add checks on every page to fetch session and user if they page reload and we lose scope and rootscope variables. See issue #42